src/basic/label.c

```
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <stddef.h>
+
+#include "label.h"
+
+static const LabelOps *label_ops = NULL;
+
+void label_ops_set(const LabelOps *ops) {
```

can we return an error here, and return EBUSY if the thing is alread set? just as a safety net because we only support one kind of labelling in parallel. (and so does the kernel, but let's better be safe than sorry)